ground/gcs/src/plugins/pathplanner/modeluavoproxy.cpp

```diff
@@ -124,6 +132,25 @@ bool ModelUavoProxy::modelToObjects()
             m_eventloop.exec();
         }
     }
+
+    for (; x < instances; x++) {
```

Comment:
could you add a comment that this runs over any pre-existing waypoints the model doesn't correspond to (or not - author's preference :D)